src/components/button-group/button-group.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import { ButtonGroup } from './button-group'
+import { Button } from '../button'
+import { Meta } from '@storybook/react'
+import { figmaDesignUrls } from '../../storybook/figma'
+
+const meta: Meta<typeof ButtonGroup> = {
+  title: 'Components/ButtonGroup',
+  component: ButtonGroup,
+}
+
+export default meta
+
+export const Default = {
+  render: () => (
+    <ButtonGroup>
+      <Button>Button 1</Button>
+      <Button>Button 2</Button>
+      <Button>Button 3</Button>
+      <Button>Button 4</Button>
+      <Button>Button 5</Button>
+    </ButtonGroup>
+  ),
+  parameters: {
+    design: {
+      type: 'figma',
+      url: figmaDesignUrls.buttonGroup,
+      allowFullscreen: true,
+    },
+  },
+}
+
+export const WithPrimaryButton = {
+  render: () => (
+    <ButtonGroup>
+      <Button>Button 1</Button>
+      <Button>Button 2</Button>
+      <Button>Button 3</Button>
+      <Button>Button 4</Button>
```

Comment:
**quibble:** I'm don't think we need the comment below to be here... until we tackle the Button work, there's no guarantee we'll change `intent` to `variant`.

Reply:
Thank you for the clarification. I was under impression that the Button component has been finalized, and that we would be transitioning from 'intent' to 'variant.' I agree that this remains uncertain until we proceed with the Button work.